cmd/clusterctl/cmd/describe_cluster_test.go

```diff
@@ -331,3 +335,33 @@ func withDeletionTimestamp(object ctrlclient.Object) {
 	now := metav1.Now()
 	object.SetDeletionTimestamp(&now)
 }
+
+type Table struct {
+	tableData []string
+}
+
+func MatchTable(expected []string) gtype.GomegaMatcher {
+	return &Table{tableData: expected}
+}
+
+func (t *Table) Match(actual interface{}) (bool, error) {
+	tableString := actual.(string)
+	tableParts := strings.Split(tableString, "\n")
+
+	for i := range t.tableData {
+		if !strings.HasPrefix(tableParts[i], t.tableData[i]) {
+			return false, nil
+		}
+	}
+	return true, nil
+}
+
+func (t *Table) FailureMessage(actual interface{}) string {
+	actualTable := strings.Split(actual.(string), "\n")
+	return fmt.Sprintf("Expected %v and received %v", actualTable, t.tableData)
+}
+
+func (t *Table) NegatedFailureMessage(actual interface{}) string {
+	actualTable := strings.Split(actual.(string), "\n")
+	return fmt.Sprintf("Expected %v and received %v", actualTable, t.tableData)
```

Comment:
```suggestion
	return fmt.Sprintf("Expected %v and received %v", t.tableData, actualTable)
```

Looks like the expected and actual values are flipped.